src/ui.text-alert/config.ts

```diff
@@ -41,9 +41,35 @@ export default /*tw*/ {
     },
   },
   description: "",
-  text: "{UText}",
-  closeButton: "{UButton}",
-  closeIcon: "{UIcon}",
+  text: {
+    base: "{UText}",
+    defaults: {
+      size: {
+        xs: "sm",
+        sm: "sm",
+        md: "md",
+        lg: "lg",
+      },
+    },
+  },
+  closeButton: {
+    base: "{UButton}",
+    compoundVariants: [
+      { color: "grayscale", variant: "primary", class: "text-white hover:text-white active:text-white focus:text-white" },
+      { color: "gray", variant: "primary", class: "text-white hover:text-white active:text-white focus:text-white" },
```

Comment:
new compondVarinats... bug fix?

Reply:
just set prop `color=inherit`

Reply:
It was already set in the component, but for gray / grayscale we don't need to inherit color. Vice versa, the icon color would better to be white, in order to be seen clearly.

![image](https://github.com/user-attachments/assets/8235a9f7-848d-4060-acd9-38ea8db4c84e)